src/main.rs

```diff
@@ -14,8 +15,17 @@ struct Args {
   #[arg(short, long)]
   pub verbose: bool,
 
-  #[arg(short = 'w', help = "Skip compilation warnings")]
-  pub skip_warnings: bool,
+  #[arg(
+    short = 'W',
+    value_delimiter = ' ',
+    action = clap::ArgAction::Append,
+    long_help = r#"Skip compilation warnings
+    all, unused-defs, match-only-vars"#
+  )]
+  pub skip_warnings: Vec<String>,
+
+  #[arg(long, help = "Stops if any warning was produced")]
+  pub errors: bool,
```

Comment:
I really don't like this as a cli arg, i would prefer a more descriptive name

Reply:
I don't like it too. But what name?